working_with_health_tool/main.bal

```diff
@@ -0,0 +1,54 @@
+import ballerina/http;
+import ballerina/log;
+import ballerinax/health.fhir.r4;
+import healthcare_samples/carinbb_package as carinbb;
+
+public type Patient record {
+    string id;
+    string lastName?;
+    string firstName;
+    string middleName;
+    string gender?;
+};
+
+service / on new http:Listener(9090) {
+
+    isolated resource function get [string fhirType]/[string id]() returns @http:Payload {
+        mediaType: [
+            "application/fhir+json",
+            "application/fhir+xml"
```

Comment:
remove the fhir+xml

Reply:
fixed in [d63567e](https://github.com/ballerina-guides/healthcare-samples/pull/15/commits/d63567eabcafd01898407fb964bd003295982e05)

Reply:
Resolving as per the above comment.